src/integration.test.ts

```diff
@@ -16,7 +16,10 @@ const getKey = (paths: string[], key: string): string => {
 };
 
 describe("Integration Test", (): void => {
-  const EXEC_OPTIONS = { shell: "/usr/bin/bash" };
+  const execOptions =
+    process.platform === "linux"
+      ? { shell: "/usr/bin/bash" }
+      : { shell: "C:\\Program Files\\Git\\bin\\bash.exe" };
```

Comment:
Nit (multiple instances): I believe you can branch within the object rather than outside it to reduce duplication. Also, the Windows path definitely will never work on any other platform, so I would compare `process.platform` against `"win32"` rather than `"linux"`. The code as written would break macOS development environments for example.